src/components/Home.jsx

```diff
@@ -5,7 +5,14 @@ import './styles/home.css';
 const Home = () => (
   <div id="home" className="restrict-width">
     <h1>{'Airborne'}</h1>
-    <p>{'World\'s first autonomous drone delivery system.'}</p>
+    <p>{'World\'s first autonomous drone delivery routing system.'}</p>
+    <h2>{'Instructions'}</h2>
+    <ol>
+      <li>{'Enter pickup and drop-off location address into the search bars in \'Send\' tab.'}</li>
+      <li>{'Click \'Search\' to route delivery. Confirm that the path is OK.'}</li>
```

Comment:
@bill-feng This might be only me, so if it's not necessary, ignore this :)
How about putting example for location address like this?
Enter pickup and drop-off location address`(eg.Moto-Azabu 3-1-35 Moto-Azabu Minato Ku Tokyo)` into the search bars in 'Send' tab.

Reply:
Thanks, I updated! :)